system/mount.py

```diff
@@ -358,13 +358,17 @@ def main():
                     res,msg = mount(module, **args)
             elif 'bind' in args.get('opts', []):
                 changed = True
-                cmd = 'mount -l'
+                cmd = 'findmnt -rn %s' % args['name']
```

Comment:
We are not sure which platforms are affected by this issue. It would be good to fix it now for Linux and wait for people to report the issue on other platforms.

@jupeter Please could amend the code to make the `findmnt` used only on Linux (fall back to `mount` if `findmnt` doesn't exists or returns unexpected results)?